src/utils/calculateClicks.js

```diff
@@ -0,0 +1,17 @@
+import _ from "lodash";
+
+const calculateClicks = (campaignOrChannel, data) => {
+  const receivedData = data;
```

Comment:
this assignment is not necessary, directly use `data` with `filter`

Reply:
a95a142a96ba9ec99ee2fc325f65cccf17c148cc